feature-libs/cart/main/occ/adapters/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './converters/index';
+export * from './occ-cart-entry.adapter';
+export * from './occ-cart-voucher.adapter';
+export * from './occ-cart.adapter';
```

Comment:
missing `occ-save-cart.adapter`?

Reply:
I didn't leave it out with a purpose.  I copied [the index from spartacus/core](https://github.com/SAP/spartacus/blob/develop/projects/core/src/occ/adapters/cart/index.ts) and removed the module because it's elsewhere now.

Do you think we should add it?

Reply:
I found this:
```
/**
 * TODO(#11938): 4.0: use OccSavedCartAdapter instead from saved-cart feature-lib of cart
 *
 * When the saved cart feature module exist, there exist a race condition between the 'saveCart' key for the default occ config
 * Allows wishlist to function when the key value is pulled from the saved-cart feature-lib
 *
 * Keep in mind that this adapter will be removed in 4.0 as we will delegate this functionality to the saved-cart feature-lib
 **/
```

Reply:
So, let keep it same as what in core.

Reply:
Ah!  Good find.